packages/app/public/assets/blockchains/ethereum/tokenlist.json

```diff
@@ -0,0 +1,91 @@
+
+[
+  {
+    "address": "0x6cAcDB97e3fC8136805a9E7c342d866ab77D0957",
+    "name": "Swapr on Ethereum mainnet",
+    "symbol": "SWPR",
+    "decimals": 18,
+    "logoURI":
+      "/assets/images/tokens/swapr.png"
```

Comment:
Why should we have the images locally? Can't we use some link URI?

Reply:
Those images were hand curated. No list has good images for Swapr, XDAI, COW and some other tokens.
Having our own images to, what we consider, top tokens is the best way to have control over the design.